packages/table/src/headers/columnHeader.tsx

```diff
@@ -211,7 +211,7 @@ export class ColumnHeader extends React.Component<IColumnHeaderProps, IColumnHea
         const cell = cellRenderer(columnIndex);
         const className = classNames(cell.props.className, extremaClasses, {
```

Comment:
can you move `props.className` to be the last arg? this allows the user to override any of our computed props.